### DISCUSSION TITLE
Doubt

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#276] [Q&A] [answered]
For how many seconds do I have to run peridigm/examples/tensile_test.yaml to achieve fracture?

**Comment by djlittl** [comment, marked as answer]
The tensile_test example problem does not fracture, it just stretches.  It's a quasi-static simulation, with is inconsistent with dynamic crack growth.  You could argue that stable crack growth could be possible in this simulation, but the quasi-static solver isn't well equipped to handle that.  I.e., if you turn on bond breaking for the tensile_test simulation, the solver will probably fail to converge, and at the very least you'd need much much smaller load steps.

This simulation take a few minutes to run on my laptop using four MPI ranks.  You should see progress printed to stdout as the simulation runs.

**Reply by Nikhil23121998** [reply]
Ok, Thankyou sir
